Generate_Client/generate.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu
+
+origin="$(pwd)"
+cd ~
```

Comment:
This is unlikely to be properly portable and will depend too much on the indivudial's environment.